authentication-typescript/template/src/components/UserMenuExample.tsx

```diff
@@ -0,0 +1,117 @@
+import { InfoListItemProps, UserMenu } from '@brightlayer-ui/react-native-components';
```

Comment:
This component is renamed as a `UserMenuComponent` in the master branch of react-native-workflow. Can we recheck all the files again? I think the templates are not up to date.